index.html

```diff
@@ -240,15 +240,30 @@ <h1>Biometrics</h1>
     <h3> <span class="glyphicon glyphicon-exclamation-sign"></span> Legal and Privacy Concerns</h3>
-    <p> The secure storage of biometric data is increasingly important as biometrics such as fingerprints are used for authentication purposes. In Illinois we have the <a href="http://www.ilga.gov/legislation/ilcs/ilcs3.asp?ActID=3004&ChapterID=57">Biometric Information Privacy Act (BIPA)</a> which regulates the collection of biometric data, by required consent as well as limits on how long the data can be stored. There are currently <a href="http://www.biometricupdate.com/201508/states-considering-biometrics-capture-laws-may-look-to-illinois-privacy-laws">lawsuits</a> against companies like Facebook and Shutterfly for collecing biometric information for commercial purposes in violation of this act. Unfortunately, law enforcement may not be regulated by BIPA. </p>
+    <p>Illinois' Biometric Information Protection Act (BIPA), passed in 2008, is one of the strongest biometric privacy laws in the country, requiring companies to obtain written consent from an individual to collect their biometric data, and also to store and dispose of that data securely. In 2019 LPL and our partner organizations helped win a supreme court case to <a href="https://lucyparsonslabs.com/posts/BIPA-Victory/">protect</a> BIPA. While private companies like Clearview and Facebook, which was recently <a href="https://www.courthousenews.com/judge-approves-650-million-settlement-in-facebook-biometric-case/">sued</a> for $650 million, are subject to BIPA, public agencies such as the police are not.</p>
+    <p>Civil rights and privacy activists have raised numerous concerns around facial recognition technology, one being its accuracy. Studies have found that most facial recognition algorithms are<a href="https://www.nist.gov/news-events/news/2019/12/nist-study-evaluates-effects-race-age-sex-face-recognition-software"> significantly worse</a> at identifying people of color than white people. There have already been cases of African American men being <a href="https://www.washingtonpost.com/opinions/2020/06/24/i-was-wrongfully-arrested-because-facial-recognition-why-are-police-allowed-use-this-technology/">falsely identified</a> by police facial recognition algorithms. </p>
+    <p>In addition, Chicago’s use of facial recognition is particularly concerning due to its secrecy. It is unclear how CPD has been using the technology. CPD issued a<a href="http://directives.chicagopolice.org/directives/data/a7a57b38-140a7462-10914-0a74-6497bf3eec2deb9c.pdf?hl=true"> directive</a> for facial recognition technology in 2013 and hasn't updated it since, even though they<a href="https://www.documentcloud.org/documents/6174796-Facial-Recognition-Response.html#document/p177"> promised</a> to in 2016. The fact that this technology is primarily deployed out of the CPIC, which is quite opaque and collaborates with federal authorities, does not help. </p>
```

Comment:
I think you can collapse this sentence into the previous one. `It is unclear how CPD has been using the technology`

Reply:
They are actually required by federal law to have an up to date privacy policy. They just ignore the directive.